CHANGELOG.asciidoc

```diff
@@ -208,6 +208,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...v7.0.0-alpha2[Check the
 - Fix panic on docker healthcheck collection on dockers without healthchecks. {pull}9171[9171]
 - Fix issue with not collecting Elasticsearch cross-cluster replication stats correctly. {pull}9179[9179]
 - The `node.name` field in the `elasticsearch/node` metricset now correctly reports the Elasticsarch node name. Previously this field was incorrectly reporting the node ID instead. {pull}9209[9209]
+- Fix access to nil pointer in vsphere module {pull}9784[9784]
```

Comment:
```suggestion
- Fix panics in vsphere module when certain values where not returned by the API. {pull}9784[9784]
```

Reply:
Could you move this into https://github.com/elastic/beats/blob/master/CHANGELOG.next.asciidoc which was just introduce in the last hours? This will make it easier to have new things in the correct release in the changelog.

Reply:
Moved with the new description provided